server/internal/server/api_keys.go

```diff
@@ -45,15 +44,24 @@ func (s *S) CreateAPIKey(
 		return nil, status.Errorf(codes.AlreadyExists, "api key %q already exists", req.Name)
 	}
 
+	trackID, err := generateRandomString("key_", 16)
```

Comment:
```suggestion
	trackID, err := generateRandomString("key-", 16)
```

?

Btw, i'm thinking of using https://pkg.go.dev/k8s.io/apimachinery/pkg/util/rand#SafeEncodeString, which is currently used in https://github.com/llm-operator/model-manager/blob/main/server/internal/server/models.go#L310

Reply:
This prefix is align with OpenAI I'd format. I'm not sure but only this ID and project ID are using underscore.

Reply:
I see..